source/clear-linux/tutorials/dlrs/dlrs.rst

```diff
@@ -127,12 +142,11 @@ PyTorch single and multi-node benchmarks
 ****************************************
 
 This section describes running the `PyTorch benchmarks`_ for Caffe2 in
-single node.  We will be looking at validating the Caffe2 APIs with the
+single node. We will be looking at validating the Caffe2 APIs with the
```

Comment:
@bd-dean  Wanted to confirm that this "forward-looking" statement is OK (unchanged from original). Often we remove these types of sentences to avoid confusion about what is currently supported/tested.

Reply:
I am fine with removing this -- once we do validate the Caffe2 APIs we can add the results back in (probably v4 release).